contents/handbook/strategy/strategy.md

```diff
@@ -5,31 +5,38 @@ showTitle: true
 ---
 
 ## TL;DR
+
 Our mission is to “Increase the number of successful products in the world”.
 
 We started by building an open-source product analytics platform with all the basic things you'd need to understand user behavior - funnels, trends, session recording, feature flags and many more features. Within a year we had thousands of customers using us and we started generating revenue.
 
-We focused on our paid product earlier this year. We quickly hit a milestone for the first 5 reference customers, and we've kept selling since. The next step on our journey is to figure out how to accelerate our top of funnel growth, whilst achieving larger order values from our Enterprise product.
+We focused on PostHog Scale (our first self-hosted paid product) earlier this year. We quickly hit a milestone for the first 5 reference customers, and we've kept selling since - exclusively focussing on inbound organic demand. We've had best in class net revenue retention from these customers and our PostHog Cloud users.
+
+The next step on our journey is to figure out how to accelerate our top of funnel growth, whilst achieving larger order values from our Enterprise product.
 
 ### Context
 
 We’ve grown a lot (10s of thousands of sign ups), but it’s clear that the many of our users use us *despite* a bad experience - complex deployment and maintenance, in particular. This is a great problem to have. It means we’re solving a hair on fire problem - product analytics that you can self host.
 
-For any company, nothing matters more than product market fit. If we get that right, it’ll be much easier for customer success, marketing or sales teams to succeed.
+Whilst PostHog's adoption is driven by one developer through our open source product, we're seeing those deployments converting steadily into paying relationships with PostHog Scale.
+
+We're aiming to keep expanding this adoption within the largest organizations using PostHog, whilst scaling our community growth.
 
 ### Mission
 
 **_“Increase the number of successful products in the world”_**
 
-### Long-term vision (for 2023)
+### Long-term vision (for 2026)
+
+**_“PostHog IPOs, becoming a standard tool for building a more successful product without losing control of user data”_**
 
-**_“Everyone building a product has a clear path to making it successful without losing control of their data”_**
+An IPO is a specific outcome that strongly signals we've become a standard tool.
```

Comment:
Is it overly pedantic if I suggest we say 'go public' instead of IPO? (In case we want to aim for a direct listing or whatever)

Reply:
Nope, that's fair.